syncd/syncd_saiswitch.cpp

```diff
@@ -937,8 +937,9 @@ void SaiSwitch::helperDiscover()
 
     {
         SWSS_LOG_TIMER("discover");
-
+        set_sai_api_log_min_prio("SAI_LOG_LEVEL_CRITICAL");
```

Comment:
why this is critical ?

Reply:
because otherwise sai will dump to syslog many thousands of the error messages "something not found" when syncd discover sai api". Output to syslog seriously hits cpu which increases fast-reboot time

Reply:
there is nothing that produces critical message, so nothing would be logged except maybe something from internal sai
i would propose to log SAI_STATUS_NOT_IMPLEMENTED (which is the most errors probably) as info message in discovery method, instead of disabling all messages in this way

Reply:
Hi Kamil,

Yes. I made that level CRITICAL because no messages are produced by SAI on syncd discovery.
I had a patch making SAI_STATUS_NOT_IMPLEMENTED as INFO in SAI. I'm not sure it's still there.

Reply:
yea that would be better approach, so other meaningfull errors if present they still would be logged